src/main/java/bio/terra/testrunner/runner/TestRunner.java

```diff
@@ -35,79 +33,25 @@
   private ThreadPoolExecutor disruptionThreadPool;
   private List<List<Future<UserJourneyResult>>> userJourneyFutureLists;
 
+  // test run outputs
   private VersionScriptResult versionScriptResult;
   private List<TestScriptResult> testScriptResults;
   protected TestRunSummary summary;
+  protected TestRunFullOutput runFullOutput;
```

Comment:
This is confusing to me because now we're tracking output in multiple places (e.g. `testScriptResults` and `runFullOutput.testScriptResults`). Instead, could we just create a `TestRunFullOutput` object right before we write out the file? i.e. in `writeOutResults` method, call the constructor and pass in all the relevant `TestRunner` properties: `new TestRunFullOutput(config, summary, testScriptResults, versionScriptResult)`.

Then in `TestRunFullOutput` POJO class, you can unpack those or reformat them however we want for the full output format. In that case, I think you would also not need to have `TestRunFullOutput` extend `TestRunSummary`. They can be completely separate serialization formats.:
```
public class TestRunFullOutput {
  public TestConfiguration testConfiguration;
  public List<TestScriptResult> testScriptResults;
  public VersionScriptResult versionScriptResult;
    public String id;
    public long startTime;
    ...(and other properties)...

  public TestRunFullOutput(
      TestConfiguration testConfiguration,
      TestRunSummary testRunSummary,
      List<TestScriptResult> testScriptResults,
      VersionScriptResult versionScriptResult) {
    this.testConfiguration = testConfiguration;
    this.testScriptResults = testScriptResults;
    this.versionScriptResult = versionScriptResult;

    // unpack the summary object
    this.id = summary.id;
    this.startTime = summary.startTime;
    this.testScriptResultSummaries = summary.testScriptResultSummaries;
    ...(and other properties)...
  }
}
```

Or another way of looking at it -- let's have `TestRunFullOutput` be strictly a serialization format. No need to use it internally in `TestRunner`, just when we write output files.

Reply:
I forgo the use of `@JsonView` since it appears to create quite a bit of confusion in this PR. I removed the @JsonView annotation and instead copy the `TestRunSummary` properties directly inside `TestRunFullOutput`.